### PR TITLE
iOS updates

### PR DESCRIPTION
**Pull request by sfshaza2**
Don't land yet.

cc @sethladd @jmagman 

**Staged:** https://flutter-docs-prod--pr11140-ios-nj1tdscm.web.app/platform-integration/ios/ios-latest

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by sfshaza2** [comment]
@jmagman @sethladd, I'd like to land this unless it has any egregious errors.

**Comment by jmagman** [comment]
> @jmagman @sethladd, I'd like to land this unless it has any egregious errors.

It still has placeholder text for the Cupertino widgets.

**Comment by sfshaza2** [comment]
@jmagman, I've removed the placeholder text. we can refine the actual text later

File: src/content/platform-integration/ios/ios-latest.md
Comment (jmagman) [inline src/content/platform-integration/ios/ios-latest.md:27]: This is still placeholder text.

 I don't think this chart is quite ready, for example "should just work" doesn't sound very confident, I don't particularly want to frame it like this.

I definitely want to land removing the "beta" references on the other two pages, but can we hold off landing this new page until we've refined the wording?

**Comment by parlough** [comment]
Just noting I pulled out the other beta/release note updates to https://github.com/flutter/website/pull/11160 and updated this PR with those changes from `main`, so this PR can focus on the new page.

**Comment by sethladd** [comment]
Hi all, it looks like we're not quite ready to land this page as-is (there's still some placeholder text, and there's some phrasing we want to tweak). I see https://docs.flutter.dev/platform-integration/ios/ios-latest is live. Should we un-publish this page, or make a fast-follow fix?

**Comment by sethladd** [comment]
Looks like there were updates after this PR landed, thank you!

**Pull request merged**
